DAOS-14533 test: Update snapshot harasser

mjean308 opened the pull request:
Skip-unit-tests: true
Skip-fault-injection-test: true
Test-tag: soak_smoke

Use get_container instead of using TestContainer

Required-githooks: true

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate owners.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

mjean308 committed:
DAOS-14533 test: Update snapshot harasser

Skip-unit-tests: true
Skip-fault-injection-test: true
Test-tag: soak_smoke

Use get_container instead of using TestContainer

Required-githooks: true

Signed-off-by: Maureen Jean <maureen.jean@intel.com>

mjean308 commented:
Needs to be tested manually

daltonbohning commented on src/tests/ftest/util/soak_utils.py:
```diff
@@ -333,16 +333,13 @@ def launch_snapshot(self, pool, name):
-    container.get_params(self)
-    container.create()
-    container.open()
+    add_containers(self, pool, path='/run/container_reserved/*')
+    container = self.container[-1]
```
The previous import is now unused
> GitHub Actions / Flake8 check
>
> F401 'test_utils_container.TestContainer' imported but unused

daltonbohning commented on src/tests/ftest/util/soak_utils.py:
```diff
@@ -333,16 +333,13 @@ def launch_snapshot(self, pool, name):
+    container = self.container[-1]
     obj_cls = self.params.get(
-        "object_class", '/run/container_reserved/*')
+        "file_class", '/run/container_reserved/*')
```
Should this be `file_oclass`?

mjean308 replied:
yep.. thanks for catching that...

mjean308 committed:
use unique container params for snapshot

Skip-unit-tests: true
Skip-fault-injection-test: true
Test-tag: soak_smoke

Required-githooks: true

Signed-off-by: Maureen Jean <maureen.jean@intel.com>

mjean308 committed:
changes to oclass

Required-githooks: true

Signed-off-by: Maureen Jean <maureen.jean@intel.com>

mjean308 committed:
updates to snapshot

Skip-unit-tests: true
Skip-fault-injection-test: true
Test-tag: soak_smoke

Required-githooks: true

Signed-off-by: Maureen Jean <maureen.jean@intel.com>

mjean308 committed:
Merge branch 'master' into mjean/DAOS-14533

Skip-unit-tests: true
Skip-fault-injection-test: true
Test-tag: soak_smoke

Required-githooks: true

mjean308 committed:
change method to API

Skip-unit-tests: true
Skip-fault-injection-test: true
Test-tag: soak_smoke

Required-githooks: true

Signed-off-by: Maureen Jean <maureen.jean@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-13794/6/execution/node/904/log
